### PR TITLE
feat: apprise notify on error

### PR DESCRIPTION
**Pull request by jackblk**
* for https://github.com/Revadike/epicgames-freebies-claimer/issues/151
* fix wrong writing path for device_auths.json

### CODE REVIEW

File: claimer.js [hunk 1]
```diff
@@ -133,23 +140,34 @@ function sleep(delay) {
...
 
             History[email] = claimedPromos;
-            notify(appriseUrl, newlyClaimedPromos);
+
+            // Setting up notification message for current account
+            if (newlyClaimedPromos.length > 0) {
+                notificationMessages.push(`${email} claimed ${newlyClaimedPromos.length} freebies: ${
+                    newlyClaimedPromos.join(", ")}`);
+            } else {
+                notificationMessages.push(`${email} has no unclaimed freebies`);
```
Comment (Revadike) [inline claimer.js:164]: This may not entirely be accurate. It could be there are unclaimed freebies, but it failed to claim it.

**Comment by Revadike** [comment]
Looks good, except that one small thing mentioned above.

File: claimer.js (hunk 1 above)
Comment (jackblk) [reply claimer.js:164]: I see. I have logged the failed claim in line 143, so maybe this line will be changed to: `notificationMessages.push(`${email} has claimed 0 freebies`);`

What do you think?

Comment (Revadike) [reply claimer.js:164]: Yeah, that should do it.

**Comment by jackblk** [comment]
@Revadike PR updated

**Pull request merged**